### PR TITLE
fix missing published_at in getNonLocalizedFields route

### PR DESCRIPTION
**Pull request by petersg83**

### CODE REVIEW

File: packages/strapi-plugin-i18n/controllers/content-types.js [hunk 1]
```diff
@@ -50,7 +53,9 @@ module.exports = {
 
     ctx.body = {
       nonLocalizedFields: sanitizedNonLocalizedFields,
```
Comment (alexandrebodin) [inline packages/strapi-plugin-i18n/controllers/content-types.js:55]: unrelated but that would have been more consistent to call that nonoLocalizedAttributes

File: packages/strapi-plugin-i18n/controllers/content-types.js [hunk 2]
```diff
@@ -50,7 +53,9 @@ module.exports = {
...
-      localizations: entity.localizations.concat(pick(['id', 'locale'], entity)),
+      localizations: entity.localizations.concat(
```
Comment (alexandrebodin) [inline packages/strapi-plugin-i18n/controllers/content-types.js:56]: Any reason not to return the localizations directly as they only contain the properties you are selecting anyway ?

File: packages/strapi-plugin-i18n/controllers/content-types.js (hunk 1 above)
Comment (petersg83) [reply packages/strapi-plugin-i18n/controllers/content-types.js:55]: Yes, but the route is called `fields` too. We started 2 different names and should refactor everywhere

File: packages/strapi-plugin-i18n/controllers/content-types.js (hunk 2 above)
Comment (petersg83) [reply packages/strapi-plugin-i18n/controllers/content-types.js:56]: There is a `concat` actually, here all the locales are set not only "other locales"

**Review by alexandrebodin — APPROVED** [review]

**Pull request merged**